### PR TITLE
Show QR code in fullscreen

### PR DESCRIPTION
**Pull request by brianebeling**
Previous PR: https://github.com/corona-warn-app/cwa-website/pull/3168

"after clicking create button to create qr code, the mouse become a pointer cursor and the qr preview can be clicked to enter full screen mode. You can exit by clicking esc button or by clicking "back" on mobile phone." - [przybciu](https://github.com/przybciu)

This PR will exit draft state after https://github.com/corona-warn-app/cwa-website/pull/3168#issuecomment-1293090520 has been implemented.

---
Internal Tracking ID: [EXPOSUREAPP-14227](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14227)

### CODE REVIEW

**Comment by Ein-Tim** [comment]
On which environments was this PR tested?

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> On which environments was this PR tested?

My understanding is that this is still a Work-in-Progress. You will probably need to wait for @brianebeling to add his improvements before testing again on Safari.

File: e [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+* [32mQR-code-fullscreen[m
+  center_tag-refactoring[m
+  change-of-syntax[m
+  faq-backup-updaten[m
+  faq-updat-backup[m
+  format_option_for_downloads[m
+  master[m
+  syntax-chage[m
+  testes[m
+  your-new-Branch[m
```
Comment (MikeMcC399) [inline e:12]: @MohamedZaghdoudi 

This file should not be part of the PR.

**Comment by dsarkar** [comment]
@Ein-Tim We haven been testing on different devices and browsers. Good news: Android works in general, iOS depends: on a newer iPad it works, but on a newer iPhone it does not. Table will be published here soon. Given that the normal use case are tabled for this functionality, I suggest we could merge. Maybe add a small sentence below to tap on the QR code for fullscreen.

**Comment by brianebeling** [comment]
Can't review my own PR - but LGTM, tested locally. It was tested on different devices as well, confirming the table.

**Review by larswmh — APPROVED** [review]
